### DISCUSSION TITLE
双滑块按钮显示不了两个值

### DISCUSSION THREAD

**Discussion by hujianping110** [youzan/vant#8573] [Q&A]
双滑块按钮显示不了两个值

**Comment by chenjiahan** [comment]
双滑块模式时，value 必须是数组格式喔

**Comment by hujianping110** [comment]
------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "youzan/vant"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年6月12日(星期六) 下午4:14
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [youzan/vant] 双滑块按钮显示不了两个值 (#8573)





 
双滑块模式时，value 必须是数组格式喔
